client/state/plugins/installed/selectors-ts.ts

```diff
@@ -12,7 +12,6 @@ import type {
 	Plugin,
 	PluginFilter,
 	PluginSite,
-	PluginSites,
```

Comment:
I guess this was the sole external usage of that type outside of its type definition; that means it could now be inlined and never exported in the first place.

Reply:
It's still a public type though, used by other public types. There's no harm in exporting it.

Reply:
My understanding was that the sole remaining usage was inlineable. Note that these types were introduced as part of this TS refactor so I believe keeping this type is unnecessary.

Reply:
I don't know, I must admit I like `PluginSites` as it is 🙂 If it was just a `PluginSite[]` array, I'd inline it, but it's a keyed object.

Reply:
I disagree since these extra types mostly add noise, but happy to leave it to your judgment 😉